src/features/lightspeed/lightspeedUser.ts

```diff
@@ -332,6 +286,67 @@ export class LightspeedUser {
     this._userType = undefined;
   }
 
+  public async updateUserInformation(): Promise<void> {
+    if (this._session) {
+      await this._updateUserInformation(false, this._session);
+    }
+  }
+
+  private async _updateUserInformation(
+    createIfNone: boolean,
+    session: vscode.AuthenticationSession,
+  ): Promise<boolean> {
+    try {
+      const userinfo: LoggedInUserInfo = await this.getUserInfo(
+        session.accessToken,
+      );
+      this._session = session;
+
+      let markdownUserInfo: string = "";
```

Comment:
I believe we can avoid the try inside an existing try block by moving the block above.

Reply:
I also do not like nested try block, but calling getUserInfo before calling getUserInfoFromMarkdown seems more natural than the reversed order... Let me merge this version and consider refactoring later.